### DISCUSSION TITLE
Use the REST API with the official source

### DISCUSSION THREAD

**Discussion by mcosta74** [microsoft/winget-cli-restsource#200] [Q&A]
Hello all, 

Is it possible to query the official `msstore` source with the REST API? (I mean not using neither the `winget-cli` and then `PowerShell` module.

For instance I'd like to have a script (using the `cURL` program) that fetches information about a specific  package and run it from a Linux machine.

Which URL should I use? Request headers ... ?

**Comment by RDMacLachlan** [comment]
Hello @mcosta74 , can you provide some more details on your request. 

Thanks,
Roy
